### ISSUE TITLE
certain date/time text should be text-selectable

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#5628]
There's no reason some of the text within the calendar views shouldn't be text-selectable with a mouse cursor. Such as...

- list view: https://fullcalendar.io/docs/list-view-demo
  - time text
  - event title text
  - day header text
- month view: https://codepen.io/arshaw/pen/PoZXmjv?editable=true&editors=001
  - day number text (when cells are not selectable)
- timegrid view: https://fullcalendar.io/docs/timegrid-standard-view-demo
  - all-day text
  - time slot text

**Comment by arshaw** [comment]
Most of the text here is not selectable because FullCalendar is preparing for a drag and cancels the default behavior on mousedown.

**Comment by ewarrenG** [comment]
Any update here? I'd like to be able to select/highlight the text associated with an event with my cursor as I'd expect other text on webpage...

**Comment by arshaw** [comment]
A fix has been implemented in [v7.0.0-beta.4](https://github.com/fullcalendar/fullcalendar-workspace/releases/tag/v7.0.0-beta.4)

Updated repro that shows all 3 views (plz use view switcher at top right):
https://codepen.io/arshaw/pen/bNbLroX?editors=0010
